src/Bicep.Core/Registry/ModuleDispatcher.cs

```diff
@@ -116,7 +116,6 @@ public ResultWithDiagnosticBuilder<ArtifactReference> TryGetArtifactReference(IA
             ModuleDeclarationSyntax => x => x.ModulePathHasNotBeenSpecified(),
             DeployDeclarationSyntax => x => x.ModulePathHasNotBeenSpecified(),
             TestDeclarationSyntax => x => x.PathHasNotBeenSpecified(),
-            DeployDeclarationSyntax => x => x.PathHasNotBeenSpecified(),
             _ => throw new NotImplementedException($"Unexpected artifact reference syntax type '{artifactReferenceSyntax.GetType().Name}'.")
```

Comment:
Curious why `x.ModulePathHasNotBeenSpecified()` for `DeployDeclarationSyntax` and not `x.PathHasNotBeenSpecified()`?

Reply:
`PathHasNotBeenSpecified()` probably makes more sense. I meant to delete a duplicate to fix build, but didn't notice the diagnostic difference.

Reply:
Fixed.